packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -537,9 +537,8 @@ function digestAuthAxiosConfig(
 async function proxyRequestToAxios(
 	uriOrObject: string | IDataObject,
 	options?: IDataObject,
+	// tslint:disable-next-line:no-any
```

Comment:
This file is in `core` and does not belong here.

Reply:
restored from master